### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [rockbenben/LearnData#37] [Comments]
# index

本博客模板采用 LearnData ✨ 为什么建这个博客？ 曾经在网络上发现了很多很好的文章， 由于没有记录保存，很多文章都已经找寻不到踪迹了，这个博客可以提供一个收集网络资源的功能。 自己有时候要记录一些经验，或者偶尔心血来潮想随便写点东西，存储在本地的话很容易被我误删， 这个博客可以满足一下我随便乱写的欲望。 白嫖一下 github.io 的服务器...

https://luojunll.github.io/

**Comment by luojunll** [comment]
Hello, world!

**Comment by 1fantasy1** [comment]
123456
